build/ci/linux/package.sh

```diff
@@ -51,8 +51,19 @@ chmod a+rw $ARTIFACT_NAME.7z
 fi
 
 if [ "$PACKTYPE" == "appimage" ]; then
+# To enable automatic updates for AppImages, set UPDATE_INFORMATION according to
+# https://github.com/AppImage/AppImageSpec/blob/master/draft.md#update-information
+case "${BUILD_MODE}" in
+  "stable_build")  export UPDATE_INFORMATION="gh-releases-zsync|musescore|MuseScore|latest|MuseScore-*x86_64.AppImage.zsync";;
+  "nightly_build") export UPDATE_INFORMATION="zsync|https://ftp.osuosl.org/pub/musescore-nightlies/linux/3x/nightly/MuseScoreNightly-latest-x86_64.AppImage.zsync";;
+  *) unset UPDATE_INFORMATION;; # disable updates for other build modes
+esac
 bash ./build/ci/linux/tools/make_appimage.sh $INSTALL_DIR
 APPIMAGE_FILE="$(ls $BUILD_DIR/*.AppImage)"
+if [ -v UPDATE_INFORMATION ]; then
+  ZSYNC_FILE="$(ls $BUILD_DIR/*.AppImage.zsync)" # data for delta updates
+  echo "ZSYNC_FILE: $ZSYNC_FILE" # TODO: Upload to server alongside AppImage
+fi
```

Comment:
Since `BUILD_MODE=devel_build` for PRs, this `echo` doesn't appear in the build log for this PR on GitHub Actions.